test/unit/lib/aws/request.test.js

```diff
@@ -0,0 +1,401 @@
+'use strict';
```

Comment:
This should be located in `test/unit/aws/request.test.js`

Reply:
Yes, it should :smile:

Reply:
Actually this will not match the current organization in the test/unit folder right? We currently have the exact same tree in `test/unit/lib` and `lib` after that change that won't be the case anymore. @medikoo is that expected?

Reply:
ah... sorry, ignore that comment! I got confused :) All is good